Added rating event and rating information in JSON

N0ps32 opened the pull request:
This PR fixes https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/403

I added a new object called rating which has 2 fields liked and disliked that contain boolean values.

MarshallOfSound commented on src/main/utils/PlaybackAPI.js:
```diff
@@ -35,6 +42,10 @@ class PlaybackAPI {
         albumArt: null,
       },
+      rating: {
+        liked: false,
```
Why are we translating a number in the 0-5 range into 2 booleans.  If users have the 5 star experiment turned on this could be any value (1, 2, 3, 4, 5)

N0ps32 replied:
I didn't even know you could have more than like/dislike in GM.
I'll look into it.

N0ps32 replied:
I do not have this experiment maybe it has been removed?
According to this entry in the product forum it's gone for good: https://productforums.google.com/forum/#!topic/play/HwKmVF5N5YA

MarshallOfSound commented on src/main/utils/PlaybackAPI.js:
```diff
@@ -63,12 +74,25 @@ class PlaybackAPI {
+  _setRating(rating) {
+    this.data.rating.liked = rating === '5';
+    this.data.rating.disliked = rating === '1';
+    this._fire('change:rating', this.data.rating);
```
Implement this in the webSocket API as well please

MarshallOfSound commented:
Please add this to the docs for both the JSON API and the webSocket API

N0ps32 commented:
I added the documentation and implemented the websocket API. I left the liked/disliked format for now since I don't think the 5 star system exists anymore.

N0ps32 commented:
Is there a way to restart the appveyor build? It seems to have been stopped for some reason. The other ones pass though so the code should be ok.

MarshallOfSound commented:
Rebuilding now
